Modules/Models/Configuration.swift

```diff
@@ -46,6 +47,9 @@ public struct Configuration: Codable {
     case supportPhoneClosingTime = "support_phone_closing_time"
   }
 
+  /// Countries of interest map
+  public let countries: [String: [String: String]]
```

Comment:
I would change this in `[UserLanguage: Country]`. A map of this kind is very very error prone